### PR TITLE
Add dockerfile

### PR DESCRIPTION
**Pull request by tmaczukin**
This PR makes possible to use exporter as Docker container.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+dependencies:
+	go get github.com/digitalocean/godo
```
Comment (ayufan) [inline Makefile]: `go get -v ./cmd/digitalocean_exporter`?

**Review by ayufan — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM golang:1.8-alpine3.6
+
+EXPOSE 9292
+
+WORKDIR /go/src/github.com/andrewsomething/digitalocean_exporter
+
+RUN apk add -U ca-certificates
+
+COPY . .
+RUN apk add --no-cache git make; \
```
Comment (ayufan) [inline Dockerfile]:
```
go get -v ./cmd/digitalocean_exporter
```

I believe that this should also compile and install in path,

**Comment by andrewsomething** [comment]
Strange that Travis is failing here but not in general. Their docs state `go get -t ./...` should be run... Either way, I set it up to explicitly install the dependencies. Re-basing on master should make this green.

File: Dockerfile (hunk 2 above)
Comment (tmaczukin) [reply Dockerfile]: Yes, it does. And it allows us to not use Makefile. I'll update the PR.

**Comment by tmaczukin** [comment]
@andrewsomething It's green now.

**Comment by andrewsomething** [comment]
:+1:  Great! Thanks again.

**Pull request merged**